spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -29,65 +29,22 @@ def create_workspace_csv(workspaces, client_id_column)
   temp_file
 end
 
-shared_context 'release brick' do
-  before(:all) do
-    config_path = ConfigurationHelper.create_interpolated_tempfile(
-      @config_template_path,
-      @test_context
-    )
-    params = JSON.parse(File.read(config_path))
-    @brick_result = GoodData::Bricks::Pipeline.release_brick_pipeline.call(params)
-    master_ids = @brick_result[:results]['CreateSegmentMasters'].map { |r| r[:master_pid] }
-    $master_projects = master_ids.map { |id| @prod_rest_client.projects(id) }
-    $master = $master_projects.first
-    pp @brick_result
-  end
-end
-
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)